scrapers/sugarcookie.yml

```diff
@@ -0,0 +1,25 @@
+name: sugarcookie
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - sugarcookie.xxx/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //h1[@class="entry-title cb-entry-title cb-title"]/text()
+      Date:
+        selector: //div[@class="cb-entry-header cb-meta clearfix"]/div[@class="cb-byline"]/span[@class="cb-date"]/time[@class="updated"]/@datetime
+      Details: 
+        selector: //span[@class="cb-itemprop"]/p/strong/text()
```

Comment:
The `text()` function removes the anchor text from the descriptions
```yaml
        selector: //span[@class="cb-itemprop"]/p/strong
        concat: "\n\n"
```
is enough. Also `\n\n` makes it more appealing than `\n` imho